pkg/planner/core/physical_plans.go

```diff
@@ -1678,7 +1678,11 @@ func (p *PhysicalIndexJoin) Clone(newCtx base.PlanContext) (base.PhysicalPlan, e
 		return nil, err
 	}
 	cloned.basePhysicalJoin = *base
-	cloned.innerPlan, err = p.innerPlan.Clone(newCtx)
+	// after the new refactor of indexJoin, the inner plan field is not a necessary one.
```

Comment:
It's hard for other people to know what "the new refactor" means, and why it becomes unnecessary.
It's better to specifically explain why it could be `nil`, or just remove the comments.

Reply:
sure, make sense